### PR TITLE
Lets Go: Fix Sparkly Swirl

### PR DESCRIPTION
**Pull request by ImLycan**
https://replay.pokemonshowdown.com/gen7letsgoou-838193924
On turn 2 Alolan Ninetales was cured of Paralysis from Sparkly Swirl.
This should fix it so it cures the user's side

### CODE REVIEW

File: data/moves.js [hunk 1]
```diff
@@ -16016,16 +16016,18 @@ let BattleMovedex = {
...
 		flags: {protect: 1},
-		onHit: function (pokemon, source, move) {
-			this.add('-activate', source, 'move: Sparkly Swirl');
-			let success = false;
-			for (const ally of pokemon.side.pokemon) {
-				if (ally !== source && (ally.volatiles['substitute'] && !move.infiltrates)) {
-					continue;
+		self: {
+			onHit: function (pokemon, source, move) {
+				this.add('-activate', source, 'move: Sparkly Swirl');
```
Comment (Zarel) [inline data/moves.js]: ????

Comment (Zarel) [reply data/moves.js]: afaik this should be `-activate` `Aromatherapy`

File: data/moves.js [hunk 2]
```diff
@@ -16016,16 +16016,18 @@ let BattleMovedex = {
...
+				for (const ally of pokemon.side.pokemon) {
+					if (ally !== source && (ally.volatiles['substitute'] && !move.infiltrates)) {
+						continue;
+					}
+					if (ally.cureStatus()) success = true;
 				}
-				if (ally.cureStatus()) success = true;
-			}
-			return success;
+				return success;
```
Comment (Zarel) [inline data/moves.js]: Why are we returning `success` anyway?

**Comment by ImLycan** [comment]
My only change was putting a `self` so it works on the user's team. It was probably my fault anyways for not writing it better :[

**Pull request closed without merging**